doc/releases/changelog-dev.md

```diff
@@ -109,6 +112,23 @@
   RX(-1, wires=[0])
   ```
 
+* `qml.operation.expand_matrix` now supports qutrit matrices such that `Operator.matrix` is now able to permute and
+  expand qutrit matrices according to the given wire order.
+
+  ```pycon
+  >>> op = qml.TShift(wires=0)
+  >>> op.matrix(wire_order=[0, 1])
+  array([[0, 0, 0, 0, 0, 0, 1, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 1, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0, 1],
+         [1, 0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 1, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 1, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 1, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 1, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 1, 0, 0, 0]])
+    ```
+
```

Comment:
```suggestion
```
This can all be removed, since the `expand_matrix` change has been separated out.